src/chain/redb/indexes/address_payment_part.rs

```diff
@@ -0,0 +1,31 @@
+use ::redb::{MultimapTableDefinition, ReadTransaction};
+use std::hash::{DefaultHasher, Hash as _, Hasher};
+
+use crate::model::BlockSlot;
+
+type Error = crate::chain::ChainError;
+
+pub struct AddressPaymentPartApproxIndexTable;
+impl AddressPaymentPartApproxIndexTable {
+    pub const DEF: MultimapTableDefinition<'static, u64, u64> =
+        MultimapTableDefinition::new("addresspaymentpartapproxindex");
```

Comment:
I would revisit the naming convention:
- `aproxindex` suffix seems irrelevant
- `part` suffix seems irrelevant
- `address` can be inferred from `payment`
- add `by` prefix as convention to identify indexes (what we use in state store)

This would end up being `bypayment`. Happy to discuss other alternatives, but I would keep it as short as possible.